scripts/index.ts

```diff
@@ -2,6 +2,7 @@ const [bin, script, ...steps] = process.argv;
 
 import { checkInstall } from './check-install';
 import { encryptCredentials, decryptCredentials } from './credentials';
+import { getPluginTypes } from './get-plugin-types';
```

Comment:
Where is `get-plugin-types`? And why is it imported?

Reply:
Oops -- removed this

This was part of a previous commit, back when I would fetch plugin types separately from `npm install`. The most recent commit removed this script but I forgot to include this change